docs/DeveloperGuide.md

```diff
@@ -519,15 +519,21 @@ otherwise)
 - **Interaction Details**: Verifies that the interaction details are provided and are non-empty, allowing the Salesperson to record free-form notes about the interaction, such as the nature, location, or purpose.
 3. Upon successful validation, AddressBook logs the interaction details in the contact’s profile. If a date is provided, it is also stored with the log entry.
 4. AddressBook displays a success message confirming the operation:
-   ```
-   Added history to Person: William Go; Phone: 96341234; Email: william.go@example.com; Address: Blk 34 Bishan St 23, #07-45; Remark: Looking to purchase condo; Birthday: 1975-12-30; Remark: Looking to purchase condo; Tags: [buyer]; DateOfCreation: 2024-01-22; History: Date of Creation: 2024-01-22
-   [2024-02-05]:
-     [2024-02-05] Negotiated condo deal
-   [2024-03-03]:
-     [2024-03-03] Follow-up on condo purchase
```

Comment:
The date before each single message has been removed

Reply:
Fixed, thanks